app/api/csv/typeParsers.ts

```diff
@@ -0,0 +1,72 @@
+import url from 'url';
+
+import { RawEntity } from 'api/csv/entityRow.js';
+import { PropertySchema, MetadataObjectSchema } from 'shared/types/commonTypes';
+import { ensure } from 'shared/tsUtils';
+
+import geolocation from './typeParsers/geolocation';
+import multiselect from './typeParsers/multiselect';
+import select from './typeParsers/select';
+import relationship from './typeParsers/relationship';
+
+const defaultParser = () => async (
+  entityToImport: RawEntity,
+  property: PropertySchema
+): Promise<MetadataObjectSchema[]> => [{ value: entityToImport[ensure<string>(property.name)] }];
+
+export default {
```

Comment:
review if the constructor for this methods is really needed

Reply:
removed the constructor